test/integration.js

```diff
@@ -37,7 +37,7 @@ function aggrAndTick() {
 }
 
 tape('submit events and ensure they are accounted for', async function(t) {
-	const evs = genEvents(3).concat(genEvents(2, 'anotherPublisher'))
+	const evs = genEvents(3, null, null, 'legacy_300x250').concat(genEvents(2, 'anotherPublisher'))
```

Comment:
use a different string as an example ID, since this is a size and someone might get confused; use "unit_123" for example